### PR TITLE
Feature/team: Team, Member API 구현

### PR DESCRIPTION
**Pull request by WoungSub-Byun**

### CODE REVIEW

**Review by seojinsarejj — COMMENTED** [review]
You did a good job.

**Comment by seojinsarejj** [comment]
You did a good job.

File: dorandoran/config/settings/__init__.py [hunk 1]
```diff
@@ -3,4 +3,4 @@
 SETTINGS_MODULE = os.environ.get("DJANGO_SETTINGS_MODULE") 
 # 환경 변수에서 DJANGO_SETTINGS_MODULE을 지정해주지 않으면 .local 파일을 임포트 해준다
 if not SETTINGS_MODULE or SETTINGS_MODULE == "config.settings":
-    from .local import *
\ No newline at end of file
+    from .dev import *
```
Comment (hanbin8269) [inline dorandoran/config/settings/__init__.py]:
```suggestion
    from .local import *
```

File: dorandoran/team/permissions.py [hunk 2]
```diff
@@ -1,18 +1,7 @@
...
-        queryset = User.objects.filter(email=value).filter(role=2)
-        if not queryset.exists():
-            return
-        return True
-
-    def is_valid_project_name(value):
-        project_format = re.compile("(.+)[-](.+)")
-        if not project_format.search(value):
-            return False
+class isTeacherOrNotDelete(permissions.BasePermission):
```
Comment (hanbin8269) [inline dorandoran/team/permissions.py]:
```suggestion
class IsTeacherOrNotDelete(permissions.BasePermission):
```

File: dorandoran/team/views.py [hunk 3]
```diff
@@ -1,56 +1,68 @@
...
+
+from account.authentication import CustomJSONWebTokenAuthentication, jwt_get_uid_from_payload_handler
 from account.models import User
+from config.settings.dev import JWT_AUTH
+from .models import Team, LinkedTeamUser
 from .serializers import TeamSerializer, LinkedTeamUserSerializer
-from account.authentication import CustomJSONWebTokenAuthentication
-from rest_framework_jwt.authentication import JSONWebTokenAuthentication
-from rest_framework import permissions
+from .permissions import isTeacherOrNotDelete
```
Comment (hanbin8269) [inline dorandoran/team/views.py]:
```suggestion
from .permissions import IsTeacherOrNotDelete
```

File: dorandoran/team/views.py [hunk 4]
```diff
@@ -1,56 +1,68 @@
...
-        serializer = TeamSerializer(data=request.data)
-        if serializer.validate(request.data) and serializer.is_valid(request.data):
-            serializer.save()
-            return Response(serializer.data, status=200)
-
-
-class TeamRetrieveDestroyView(generics.RetrieveDestroyAPIView):
-    queryset = Team.objects.all()
-    serializer_class = TeamSerializer
+    permission_classes = [IsAuthenticated & isTeacherOrNotDelete]
```
Comment (hanbin8269) [inline dorandoran/team/views.py]:
```suggestion
    permission_classes = [IsAuthenticated & IsTeacherOrNotDelete]
```

**Review by hanbin8269 — APPROVED** [review]

**Pull request merged**